### ISSUE TITLE
COCO labels

### ISSUE THREAD

**Issue by ahundt** [PavlosMelissinos/enet-keras#19]
Do you represent each label as separate channels in the dataset loader?

I ask because there is a lot of channel overlap and the z order isn't always correct in coco. For example the table category often blocks out all the objects on top of the table if you put it all into a single categorical chanel, rather than a one-hot (multiple-hot?) encoding.

**Comment by PavlosMelissinos** [comment]
You're right about the issue, thanks. However, I've tried a couple ways to represent the ground truth. One of them is as a NxHxWxC tensor, each label being a separate channel, like you describe; so in that case, C is 80 plus an extra channel for background, which makes 81 in total.

It's [in here](https://github.com/PavlosMelissinos/enet-keras/blob/master/src/data/datasets.py#L366-L401).

**Comment by ahundt** [comment]
Cool thanks! I just wanted to confirm, sorry I missed those lines.

**Comment by ahundt** [comment]
I wonder, should the data be put in sparse tensors?

Probably would need to test to see if it is a faster or slower approach to storing the data.
